package.json

```diff
@@ -161,20 +160,20 @@
     "postinstall": "patch-package"
   },
   "dependencies": {
+    "@types/big.js": "^6.0.2",
     "@types/qs": "^6.9.5",
     "@types/uuid": "^8.3.0",
-    "@types/big.js": "^6.0.2",
-    "@walletconnect/client": "^1.3.1",
+    "@walletconnect/client": "^1.3.6",
```

Comment:
I believe this doesn't work. When I tried running this version I got errors regarding importing/exports.

Reply:
That’s the error mentioned in https://github.com/rollup/plugins/issues/762. I’ve downgraded `@rollup/plugin-commonjs` until that is fixed.

Reply:
I'd swear I had tried this new version without bumping the rollup dep and I had a lot of runtime errors. That goods look now. There is just this additional console log when using this version:

```
Circular dependency: node_modules/@pedrouid/iso-crypto/dist/cjs/helpers/index.js -> node_modules/@pedrouid/iso-crypto/dist/cjs/helpers/env/index.js -> node_modules/@pedrouid/iso-crypto/dist/cjs/helpers/env/browser.js -> node_modules/@pedrouid/iso-crypto/dist/cjs/helpers/env/fallback.js -> node_modules/@pedrouid/iso-crypto/dist/cjs/helpers/index.js
Circular dependency: node_modules/@pedrouid/iso-crypto/dist/cjs/helpers/index.js -> node_modules/@pedrouid/iso-crypto/dist/cjs/helpers/env/index.js -> node_modules/@pedrouid/iso-crypto/dist/cjs/helpers/env/browser.js -> node_modules/@pedrouid/iso-crypto/dist/cjs/helpers/env/fallback.js -> /home/nuno/radicle-upstream/node_modules/@pedrouid/iso-crypto/dist/cjs/helpers/index.js?commonjs-proxy -> node_modules/@pedrouid/iso-crypto/dist/cjs/helpers/index.js
```

I suggest that we ignore that one in `rollup.app.js` like we do with other dependencies.

Reply:
Done in https://github.com/radicle-dev/radicle-upstream/pull/1628/commits/da59e9dad757a180aee5962c58107a9d51d9964b